webapp/src/cmds.ts

```diff
@@ -31,6 +35,125 @@ export function browserDownloadDeployCoreAsync(resp: pxtc.CompileResult): Promis
             resp.userContextWindow,
             e => core.errorNotification(lf("saving file failed..."))
         );
+    } else if (pxt.appTarget.compile.useModulator) {
+        core.infoNotification(lf("Here's a tune..."));
+        let lbrEnable = false;
+        let modulationVersion = 1;
+        let audioFormat = 'wav';
+
+        let bin = ltcIhexToBinary(resp.outfiles[pxtc.BINARY_HEX]);
+
+        let hex = resp.outfiles[pxtc.BINARY_HEX];
+
+        function getCanvas(): HTMLCanvasElement {
+            return document.getElementById('wavStrip') as HTMLCanvasElement;
+        }
+        function getAudioElement(): HTMLAudioElement {
+            return document.getElementById('audio_output') as HTMLAudioElement;
```

Comment:
could we get a consistent naming for the modulator HTML DOM element, modulelatorAudioOutput...